torchbenchmark/models/llama/model.py

```diff
@@ -224,8 +224,8 @@ def forward(self, tokens: torch.Tensor, start_pos: int):
    
         h = self.tok_embeddings(tokens)
 
-        self.freqs_cis = self.freqs_cis.to(h.device)
-        freqs_cis = self.freqs_cis[start_pos : start_pos + seqlen]
+        freqs_cis = self.freqs_cis.to(h.device)
```

Comment:
Can we add a reference to https://github.com/facebookresearch/llama/pull/349 here?

Reply:
Done